wolfcrypt/src/logging.c

```diff
@@ -1215,7 +1215,7 @@ unsigned long wc_PeekErrorNodeLineData(const char **file, int *line,
 
     if (ERRQ_LOCK() != 0) {
         WOLFSSL_MSG("Lock debug mutex failed");
-        return BAD_MUTEX_E;
+        return -BAD_MUTEX_E;
```

Comment:
Needs a comment on why using double negative error value here, such as if this is required for compatibility layer API? Most of our internal api should have negative error values.... Alternative is to set 'ret' variable than fall through to where ret = -ret later on in the function.

Reply:
I think this is already fixed here: #6195

Reply:
Thanks Eric, this is indeed already fixed by your PR.
FYI Jacob, this is needed for OpenSSL compatibility.  The function returns an unsigned long, so negative values are not valid, and OpenSSL expects a positive value for errors.